free-1.md

```diff
@@ -10,41 +10,157 @@ _What need is met by your idea? Who are you helping? What is that person's
+But a tableau tree has enough distinguishing characteristics from any tree 
+diagram that a DSL for tableau trees should have a simplified interface.
+In CS 81, the construction of a tableau tree is reduced to two operations:
+stack and split. So I think a DSL can use that technique to build a simpler
+syntax for a DSL for tableau trees.
+A DSL for this would make the task much less cumbersome as hopefully users
+would find the DSL simpler to use than drawing out a tree using lines and text
+boxes.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I have to draw tableau trees for my CS 81 homework and they cannot be hand
+drawn.
+I generally TeX my homework, but I've found the syntax of packages for drawing
+trees to be difficult to understand.
+I imagine that the syntax would be even more daunting for people who do not
+use LaTeX for most of their homework.
+I TeX-ed tableau trees in the latest homework but it required a lot of trial
+and error as well as reading a _lot_ of samples of various tree structures.
+I would be really happy if there was an easier way for me to draw trees for 
+my homework.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Writing logic proofs in math.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Like LaTeX, the programmer would write text to a file which would then be
+compiled to the image of a tree.
+One possible way a program could look is
+```
+(A \or B) \and C \check
+C
+A \or B
+  A
+    branch1
+    branch2
+  B
+```
+which would produce the following tree
+
+![tree](/images/tree.png)
+
+This uses LaTeX syntax, such as backslashes to invoke certain symbols, which
+is familiar to many people who work with math.
+The structure is more easily readable for statements that stack (such as
+C and A \or B) compared to the equivalent tree in many tree packages in LaTeX
+which expects a statement on the next line to be the child of the previous
+statement (so A \or B would be the child of C) which would then draw a line
+between them. 
+This would go against tableau convention.
+The package prooftrees solves this issue but still requires A \or B to be
+indented relative to C, which is harder to read.
+
+For logicians, there should not be a need to specify math mode marked by \$
+since everything should be mathematical symbols.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program would read the entire file to calculate how much branching is
+requested. 
+It would also calculate how long each statement in the tree is based on how
+many symbols are used.
+This is then used to calculate the angle and length of lines in the tree for
+branching to give each statement enough space.
```

Comment:
What happens if you make a massive tableau? Are you willing to go off the page, or to draw something very small and/or compact? How would you approach such a tradeoff?